caffe2/contrib/mpscnn-fb/mpscnn_test.mm

```diff
@@ -11,6 +11,7 @@
 #include "caffe2/core/workspace.h"
 #include "caffe2/utils/proto_utils.h"
 
+#define CHECK_NEAR(A, B, C) assert(abs(A-B) < C);
```

Comment:
This comes from `glog`, FWIW. https://github.com/google/glog/blob/e56ed5520846cbd893944dccfbb1042d187714a6/src/glog/logging.h.in#L842

What version of glog were you using?